packages/transaction-controller/src/index.ts

```diff
@@ -22,6 +22,7 @@ export type {
   TransactionControllerTransactionSubmittedEvent,
   TransactionControllerUnapprovedTransactionAddedEvent,
   TransactionControllerMessenger,
+  TransactionControllerOptions,
```

Comment:
> FIXED: Add missing package-level export for TransactionControllerOptions type.

I think we are exporting a new type from this package more than fixing something, although it is a fix in terms of how the package is aligned with others. Thoughts on making this an **ADDED** entry?

Related to this, do you think we can edit this directly in the Unreleased section of the `CHANGELOG`?